src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -0,0 +1,280 @@
+              <Form.Control
+                onChange={e => setPolicyUri(e.target.value)}
+                placeholder="https://"
+                type="text"
+                value={policyUri}
+              />
+            </Col>
+            <Col lg={6} md={6} sm={12} xs={12}>
+              <Form.Label>Links to Terms of Service</Form.Label>
+              <Form.Control
+                onChange={e => setTosUri(e.target.value)}
+                placeholder="https://"
+                type="text"
+                value={tosUri}
+              />
+            </Col>
+          </Row>
+          {isEdit && (
+            <div className="danger">
+              <Typography
+                style={{ marginTop: '8px' }}
+                color="error"
+                variant="headline3"
+              >
+                DANGER ZONE
+              </Typography>
+              <Typography variant="smallText1">
+                Editing the following information will render your client
+                invalid and will require you to create it again and resubmit
+                verification if needed.
+              </Typography>
+              <Row>
+                <Col lg={6} md={6} sm={12} xs={12}>
+                  <Form.Label>Redirect URI(s)</Form.Label>
+                  <Form.Control
+                    required
+                    onChange={e => setRedirectUris(e.target.value)}
+                    placeholder="https://"
+                    type="text"
+                    value={redirectUris}
+                  />
+                </Col>
+                <Col lg={6} md={6} sm={12} xs={12}>
+                  <Form.Label>Sector Identifier URI</Form.Label>
+                  <Form.Control
+                    onChange={e => setSectorUri(e.target.value)}
+                    placeholder="https://"
+                    type="text"
+                    value={sectorUri}
+                  />
+                </Col>
+              </Row>
+
+              <button
+                className="delete-button"
+                onClick={() => {
+                  setIsDelete(true)
+                  setIsShowingConfirmModal(true)
+                  hide()
+                }}
+              >
+                <IconSvg options={{ icon: 'delete', color: '#f44336' }} />
+                DELETE CLIENT
+              </button>
+            </div>
+          )}
+        </Modal.Body>
+        <Modal.Footer>
+          <Button variant="default" onClick={hide}>
+            CANCEL
+          </Button>
+          <Button variant="primary" onClick={onCreateClient}>
+            Save
+          </Button>
+        </Modal.Footer>
+      </Modal>
+      <ConfirmModal
+        onClose={() => {
+          setIsShowingConfirmModal(false)
+          setIsDelete(false)
+          setSelectedClient(undefined)
+        }}
+        isDelete={isDelete}
+        showModal={isShowingConfirmModal}
+        client={isDelete ? client! : updatedClient!}
+      />
```

Comment:
IMO the design for this is unlike anything else we have in the product, so we should disregard it. Recommend just using the `WarningModal` component that we already have.